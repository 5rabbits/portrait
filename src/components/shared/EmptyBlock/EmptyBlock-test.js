import { React, toJSON, mount } from 'test/helper'
import EmptyBlock from 'shared/EmptyBlock'

describe('EmptyBlock', () => {
  it('should render with minimum props', () => {
    const component = mount(<EmptyBlock />)

    expect(toJSON(component)).toMatchSnapshot()
  })
})
