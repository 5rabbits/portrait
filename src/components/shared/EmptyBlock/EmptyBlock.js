import React from 'react'
import childrenPropType from 'propTypes/children'

const EmptyBlock = props =>
  <div className="empty-view">
    <div className="empty-container">
      <div className="empty-dialog">
        {props.children}
      </div>
    </div>
  </div>

EmptyBlock.propTypes = {
  children: childrenPropType,
}

EmptyBlock.defaultProps = {
  children: null,
}

export default EmptyBlock
