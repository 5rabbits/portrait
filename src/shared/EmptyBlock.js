import React from 'react'

const EmptyBlock = props =>
  <div className="empty-view">
    <div className="empty-container">
      <div className="empty-dialog">
        {props.children}
      </div>
    </div>
  </div>

export default EmptyBlock
