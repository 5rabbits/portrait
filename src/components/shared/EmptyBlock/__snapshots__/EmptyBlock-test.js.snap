// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyBlock should render with minimum props 1`] = `
<EmptyBlock>
  <div
    className="empty-view"
  >
    <div
      className="empty-container"
    >
      <div
        className="empty-dialog"
      />
    </div>
  </div>
</EmptyBlock>
`;
